[2023-10-02T10:23:44.930+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Load-Green-Taxi-Data-Web-To-GCS-To-Postgres.read_postgres scheduled__2021-01-02T06:00:00+00:00 [queued]>
[2023-10-02T10:23:44.945+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Load-Green-Taxi-Data-Web-To-GCS-To-Postgres.read_postgres scheduled__2021-01-02T06:00:00+00:00 [queued]>
[2023-10-02T10:23:44.945+0000] {taskinstance.py:1308} INFO - Starting attempt 2 of 3
[2023-10-02T10:23:45.041+0000] {taskinstance.py:1327} INFO - Executing <Task(PostgresOperator): read_postgres> on 2021-01-02 06:00:00+00:00
[2023-10-02T10:23:45.050+0000] {standard_task_runner.py:57} INFO - Started process 577 to run task
[2023-10-02T10:23:45.053+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'Load-Green-Taxi-Data-Web-To-GCS-To-Postgres', 'read_postgres', 'scheduled__2021-01-02T06:00:00+00:00', '--job-id', '758', '--raw', '--subdir', 'DAGS_FOLDER/web_gcs_pg.py', '--cfg-path', '/tmp/tmpcpm2ihid']
[2023-10-02T10:23:45.056+0000] {standard_task_runner.py:85} INFO - Job 758: Subtask read_postgres
[2023-10-02T10:23:45.142+0000] {task_command.py:410} INFO - Running <TaskInstance: Load-Green-Taxi-Data-Web-To-GCS-To-Postgres.read_postgres scheduled__2021-01-02T06:00:00+00:00 [running]> on host 537710aea1a2
[2023-10-02T10:23:45.265+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Load-Green-Taxi-Data-Web-To-GCS-To-Postgres' AIRFLOW_CTX_TASK_ID='read_postgres' AIRFLOW_CTX_EXECUTION_DATE='2021-01-02T06:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-01-02T06:00:00+00:00'
[2023-10-02T10:23:45.266+0000] {sql.py:265} INFO - Executing: SELECT * FROM 'public.yellow_taxi_2021-01' LIMIT 5;
[2023-10-02T10:23:45.277+0000] {base.py:73} INFO - Using connection ID 'my_postgres_conn' for task execution.
[2023-10-02T10:23:45.351+0000] {base.py:73} INFO - Using connection ID 'my_postgres_conn' for task execution.
[2023-10-02T10:23:45.355+0000] {sql.py:375} INFO - Running statement: SELECT * FROM 'public.yellow_taxi_2021-01' LIMIT 5;, parameters: None
[2023-10-02T10:23:45.356+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/operators/sql.py", line 277, in execute
    **extra_kwargs,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 349, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 380, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.SyntaxError: syntax error at or near "'public.yellow_taxi_2021-01'"
LINE 1: SELECT * FROM 'public.yellow_taxi_2021-01' LIMIT 5;
                      ^

[2023-10-02T10:23:45.366+0000] {taskinstance.py:1350} INFO - Marking task as UP_FOR_RETRY. dag_id=Load-Green-Taxi-Data-Web-To-GCS-To-Postgres, task_id=read_postgres, execution_date=20210102T060000, start_date=20231002T102344, end_date=20231002T102345
[2023-10-02T10:23:45.383+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 758 for task read_postgres (syntax error at or near "'public.yellow_taxi_2021-01'"
LINE 1: SELECT * FROM 'public.yellow_taxi_2021-01' LIMIT 5;
                      ^
; 577)
[2023-10-02T10:23:45.425+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-10-02T10:23:45.458+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
